Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pyppeteer = "*"
```

Comment:
Will man das certupload nutzen, muss man zunächst eine entsprechende Umgebung samt der Abhängigkeiten installieren. Das sollte in der Doku ergänzt werden, vielleicht vorerst nur im Teil zum certupload. Die Kommandos waren bei mir:

- pipenv per Paketmanager installieren
- pipenv shell
- pipenv install

Reply:
Ich denke die meisten nutzer würden eher `pip install puppeteer` ausführen, als ein pipenv zu verwenden. Ich nehme einfach mal beides in README auf